OxfordBodleian/Juel-Jensen/BDLaethf33.xml

```diff
@@ -17,6 +17,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 <editor key="JG"/>
                 <editor key="MV"/>
                 <editor key="SH"/>
+                <editor key="EDS"/>
```

Comment:
JK at some point?

Reply:
Yes, Jacopo, you are right. Shall we add JK to all MSS in one single PR?

Reply:
Probably a signle PR later on :)